src/System.Text.Encodings.Web/src/project.lock.json

```diff
@@ -1,18 +1,15 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
@davidfowl Should we be removing all of the lock files we have checked in? All of the projects we build today have lock files in the repository here.